modules/nf-core/trust4/main.nf

```diff
@@ -0,0 +1,73 @@
+process TRUST4 {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::trust4=1.0.13"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'biocontainers/trust4:1.0.13--h43eeafb_0':
+        'biocontainers/trust4:1.0.13--h43eeafb_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(reads)
+    path(fasta)
+    path(ref)
```

Comment:
we need meta1 and meta2 here. What is the difference between fasta and ref, how about: vdj_reference?

Reply:
ref is an additional reference required when you dont start from bam but fastq
fasta is what they call their reference basically.
I wanted to stay in line with their naming here, but am also fine with renaming them.